fluent-package/yum/systemd-test/update-to-next-version.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/commonvar.sh
+
+# Install the current
+package="/vagrant/${distribution}/${DISTRIBUTION_VERSION}/x86_64/Packages/fluent-package-[0-9]*.rpm"
+sudo $DNF install -y $package
+sudo systemctl enable --now fluentd
+systemctl status --no-pager fluentd
+
+# Make a dummy pacakge for the next version
+case $distribution in
+    amazon)
+        sudo amazon-linux-extras install -y epel
+        ;;
+    *)
+        sudo $DNF install -y epel-release
+        ;;
+esac
+sudo $DNF install -y rpmrebuild
+# Example: "Release     : 1.el9"
+release_line=$(rpm -qip $package | grep "Release*:*")
+# Example: "1"
+release_ver=$(echo $release_line | sed -E "s/.*([0-9]+)\.(.+)/\1/g")
+# Example: "2.el9"
```

Comment:
how about using rpmquery --queryformat="%{Version}"

Reply:
It will take `5.0.0`, but I want to take the package version from the `Release`.
(`1` from `1.el9`)

So I think we need to use some regexp.

Reply:
it is more simple:

`release_ver=$(echo $release | cut -d . -f1)`

Reply:
Cool! Thanks! Fixed!